src/settingslist.h

```diff
@@ -143,6 +156,7 @@ SETTING_STRING(entryServerCity,     // getter
                hasEntryServerCity,  // has
                "entryServer/city",  // key
                nullptr,             // default value
+               true,                // user setting
```

Comment:
Is there any way we could enforce this is used correctly in code? What makes a setting a user setting is it the fact that it can be invoked from QML? If so, can we make it not invokable if not a user setting? I am thinking about us and other developers in the future just copy pasting other settings and not thinking about this exact property.

Reply:
well, some values are exposed to the dev-menu. It's not so easy to implement what you are proposing.

Reply:
can we use CODEOWNERS to protect the settings list...

Reply:
As discussed on Slack, let's file a follow up to deal with this scenario as it would not be a simple change.